docs/src/terminology.md

```diff
@@ -219,6 +219,10 @@ The public key of the [account](#account) containing a [program](#program).
 
 A stack of proofs, each of which proves that some data existed before the proof was created and that a precise duration of time passed before the previous proof. Like a [VDF](#verifiable-delay-function-vdf), a Proof of History can be verified in less time than it took to produce.
 
+## prioritization fee
+
+An additional fee user can specify in compute budget [instructions](#instruction) to prioritize their [transactions](#transaction).
```

Comment:
Can we include the units in this description as well?

Reply:
yep, added more descriptions, maybe too verbose now 😃